validator/testcases/javascript/instanceproperties.py

```diff
@@ -225,6 +252,8 @@ def set_contentScript(value, traverser):
                         "is dangerous and error-prone. Please use a separate "
                         "JavaScript file, along with the "
                         "`contentScriptFile` property instead.",
+            signing_help="Please do not use the `contentScript` property "
+                         "in any add-ons submitted for automated signing.",
```

Comment:
I'm not sure. That is basically what "submitted for automated signing" means, and no-one will actually see this message if it doesn't apply to them. Except people submitting betas, I guess.